.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+WIP
+
+Includes:
+- Explain what we expect (links to CONTRIBUTING.md)
+- Screenshots, GIFs, review app (visual changes)
```

Comment:
I would explain somewhere how to add review app. Perhaps simply adding to this line "just add the label 'review up' to the PR" or something.

LGTM anyway

Reply:
I will write more than this. It's just that it's Friday afternoon, so this is more like a reminder :slightly_smiling_face: